includes/wc-template-hooks.php

```diff
@@ -180,6 +180,11 @@
 add_action( 'woocommerce_single_variation', 'woocommerce_single_variation', 10 );
 add_action( 'woocommerce_single_variation', 'woocommerce_single_variation_add_to_cart_button', 20 );
 
+/**
+ * Product attributes.
+ */
+add_filter( 'woocommerce_attribute_label', 'wp_kses_post', 100 );
```

Comment:
This is used in frontend and admin - add to wc-core-functions.php instead. There is a section commented as:

```
/**
 * Filters on data used in admin and frontend.
 */
```